src/Components/NewBeer.js

```diff
@@ -0,0 +1,110 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+import Nav from './Nav';
+
+class NewBeer extends Component {
+  state = {
+    name: '',
+    tagline: '',
+    attenuation_level: 0,
+    first_brewed: '',
+    description: '',
+    contributed_by: '',
+    brewers_tips: '',
+  };
+
+  inputChangeHandler = (event) => {
+    let { name, value } = event.target;
+    this.setState({
+      [name]: value,
+    });
+  };
+
+  handleFormSubmission = (event) => {
+    event.preventDefault();
+    axios
```

Comment:
I would recommend using destructuring for this use case, it will make your code more readable 
```
 const {name, tagline, attenuation_level, first_brewed, description, contributed_by, brewers_tips} = this.state;
    axios
      .post('https://ih-beers-api2.herokuapp.com/beers/new', {
        name,
        tagline,
        attenuation_level,
        first_brewed,
        description,
        contributed_by,
        brewers_tips,
      })